lang/sensei-lms.pot

```diff
@@ -1,12 +1,12 @@
-# Copyright (C) 2019 sensei
-# This file is distributed under the same license as the sensei package.
+# Copyright (C) 2019 sensei-lms
+# This file is distributed under the same license as the sensei-lms package.
 msgid ""
 msgstr ""
-"Project-Id-Version: sensei\n"
+"Project-Id-Version: sensei-lms\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Report-Msgid-Bugs-To: https://www.transifex.com/woothemes/sensei-by-woothemes/\n"
+"Report-Msgid-Bugs-To: https://translate.wordpress.org/projects/wp-plugins/sensei/\n"
```

Comment:
This should be `sensei-lms` as well. We'll be regenerating this file when we release though, so not a big deal.

Reply:
Ahhh yeah, I updated gulpfile but didn't regenerate.